app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
+  def create
```

Comment:
- [ ] As the previous reviewer suggested kindly make sure to create a `new` action in your `comments` controller so your app work as expected and your project got approved
`The comments controller should have new and create actions`